src/demo/helpers.js

```diff
@@ -43,11 +43,6 @@ export const bodyAnchorFromType = (body, type) => {
 
 export const colorForFishPart = (palette, part) => {
   switch (part.type) {
-    case FishBodyPart.MOUTH:
-      // If the mouth should be tinted, return the mouth
-      // color. Otherwise, return null so that it is
-      // not tinted.
-      return part.tinted ? palette.mouthRgb : null;
```

Comment:
note on this: Mark said none of the new fish mouth assets should be tinted, so I removed the `tinted` property from `fishData.mouths`